Components/avatar.css

```diff
@@ -0,0 +1,29 @@
+.avatar img{
+    border-radius: 50%;
+    object-fit: cover;
+}
+
+.avatar-xx-l img{
+    width:120px;
+    height:120px;
+}
+.avatar-x-l img{
+    width:100px;
+    height:100px;
+}
+.avatar-l img{
```

Comment:
.`avatar-l` here is **avatar-large** right? it yes then it can be named as **avatar-lg for large** and **for middle avatar-md**  and **for small avatar-sm**

Reply:
+1